### PR TITLE
[Test] Estimate time of java project build and run inside Che Code Editor in performance tests

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
This PR adds `api load tests` scripts that will be used in DevSpaces performance testing. 

### What issues does this PR fix or reference?
Openshift DevSpaces issue - [Estimate time of java project build and run inside Che Code Editor in performance tests](https://issues.redhat.com/browse/CRW-4044).

### How to test this PR?
1. Create test users on cluster using [performance-users](https://main-jenkins-csb-crwqe.apps.ocp-c1.prod.psi.redhat.com/job/Testing/job/Performance/job/performance-users-creation/) job.
2. Api load test starts by command like:
```
./api-load-tests.sh -u user  -p load-user -s lombok -n 11 -c 1 -b https://devspaces.crw-qe.com -o https://api.ocp410.com:6443
    -u base username 
    -p common password 
    -b URL of DevSpaces 
    -o Openshift cluster URL
    -s test suite (by default its php)
    -n number of users ${NC} usernames will be set in format <username>1, <username>2(by default its 1) 
    -c count on how many times one user should run a workspace(by default its 1)
```
For now `php`, `cpp`, `python`, `lombok` and `quarkus` suites are available for testing.

3. Tests results adds to result.txt with format like:
```
Load testing for 11 users x 1 workspaces took 215 seconds
11 tests passed
0 tests failed
Workspace startup time: 
min: 37 second
avr: 44 second
max: 50 second
Project build time:
min: 140 second
avr: 142 second
max: 146 second
```
All logs from test pods saved in reports folder.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix or new feature ](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix-or-new-feature)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: tests/performance/api-load-tests.sh [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  echo "Parameter -t wasn't set, setting completitions count to 1."
+  export COMPLETITIONS_COUNT=1
+fi
+
+if [ -z $USER_COUNT ]; then
+  echo "Parameter -t wasn't set, setting users number to 1."
+  export USER_COUNT=1
+fi
+
+if [ -z $TEST_SUITE ]; then
```
Comment (dmytro-ndp) [inline tests/performance/api-load-tests.sh:42]: There is no "TEST_SUITE" parameter in eclipse/che e2e tests anymore.

Use -e USERSTORY=Quarkus parameter to run https://github.com/eclipse/che/blob/main/tests/e2e/specs/devfiles/Quarkus.spec.ts test

Comment (SkorikSergey) [reply tests/performance/api-load-tests.sh:42]: That is api-suite from api-test folder.

Comment (SkorikSergey) [reply tests/performance/api-load-tests.sh:42]: https://github.com/eclipse/che/tree/apiLoadTests/tests/performance/api-tests

**Review by dmytro-ndp — APPROVED** [review]

File: tests/performance/api-load-tests.sh (hunk 1 above)
Comment (dmytro-ndp) [reply tests/performance/api-load-tests.sh:42]: Got it.

**Pull request merged**